src/components/radio/radio.twig

```diff
@@ -0,0 +1,23 @@
+{% set attributes %}
+  type="radio"
+  class="c-radio{% if class %} {{ class }}{% endif %}"
+  {% for attribute_name in [
+    'value',
+    'id',
+    'name',
+    'autocomplete'
+  ] %}
+    {% set attribute_value = attribute(_context, attribute_name) %}
+    {% if attribute_value %}{{ attribute_name }}="{{ attribute_value }}"{% endif %}
+  {% endfor %}
+  {% for attribute_name in [
+    'checked',
+    'required',
+    'disabled'
+  ] %}
+    {% set attribute_value = attribute(_context, attribute_name) %}
+    {% if attribute_value %}{{ attribute_name }}{% endif %}
+  {% endfor %}
+{% endset %}
+
+<input {{ attributes }}>
```

Comment:
Part of me wonders if the complexity of this component is worth it.

Would it be better to just tell component consumers to drop a specific class on a radio input? It seems like this component could become a hindrance if we needed to add an unsupported attribute or make changes in the future.

Reply:
(Not a strong preference/blocker)

Reply:
I wondered the same thing, but since this was copy/pasted from the checkbox, the thought of downgrading that component for parity made me a bit weary.

Maybe in the future we could combine the two into one simpler component?

Reply:
@Paul-Hebert I've added a comment to both the checkbox and radio templates to acknowledge this suggestion.

Reply:
Sounds good!